pages/new-mission.vue

```diff
@@ -0,0 +1,486 @@
+              @end="dragging = false"
+            >
+              <div
+                v-for="element in list"
+                :key="element.name"
+                class="list-group-item item"
+              >
+                <b-row style="align-items: center">
+                  <b-col style="display: flex; align-content: center; align-items: center;">
+                    <img class="round--picto"
+                         :src="element.logo"
+                    >
+                    <p>{{ element.name }}</p>
+                  </b-col>
+                </b-row>
+              </div>
+            </draggable>
+          </b-col>
+        </b-row>
+
+        <b-button variant="primary" :disabled="$v.form.$invalid" @click="createMission">
+          Submit
+        </b-button>
+      </b-form>
+    </b-container>
+  </div>
+</template>
+
+<script>
+  import { validationMixin } from 'vuelidate'
+  import { url } from 'vuelidate/lib/validators'
+  import draggable from 'vuedraggable'
+  import 'vue-select/dist/vue-select.css'
+
+  let id = 1
+
+  export default {
+    components: {
+      draggable
+    },
+    mixins: [validationMixin],
+    data() {
+      return {
+        form: {
+          name: null,
+          image: null,
+          logoGroup: null,
+          logoClient: null,
+          nameGroup: null,
+          nameClient: null,
+          webSite: null,
+          context: null,
+          livrable: null,
+          atecnaPlus: null,
+          category: null,
+          type: null,
+          duration: null,
+          teamSize: null,
+          location: null,
+          environments: []
+        },
+        environments: [],
+        list: [],
+        selected: null,
+        dragging: true,
+        componentData: {
+          props: {
+            type: "transition",
+            name: "flip-list"
+          }
+        }
+      }
+    },
+    validations: {
+      form: {
+        name: {
+
+        },
+        image: {
+          url
+        },
+        logoGroup: {
+          url
+        },
+        logoClient: {
+          url
+        },
+        nameGroup: {
+
+        },
+        nameClient: {
+
+        },
+        webSite: {
+
+        },
+        context: {
+
+        },
+        livrable: {
+
+        },
+        atecnaPlus: {
+
+        },
+        category: {
+
+        },
+        type: {
+
+        },
+        duration: {
+
+        },
+        teamSize: {
+
+        },
+        location: {
+
+        },
+        environments: {
+
+        }
+      }
+    },
+    mounted() {
+      const params = {
+        token: this.$auth.getToken('local')
+      }
+
+      this.$axios.get('/api/environment/getAll', { params })
+        .then(response => {
+          console.log('response = ', response.data.environments)
+          this.environments = response.data.environments
+        })
+    },
+    methods: {
+      addToDrop: function(param) {
+        if (param.length > 0) {
```

Comment:
There is two "if" in a row, can't we regroup it or find a way to change this ?

Reply:
add the whole thing in a computed ? (or just usefull for condition in template)